choco/tools/chocolateybeforemodify.ps1

```diff
@@ -0,0 +1,6 @@
+$toolsDir = Get-ToolsLocation
+
+Write-Host "Generating extra installed packages."
+$installed_packages_command = "/C `"$toolsdir\TinyTeX\bin\win32\tlmgr.bat info --list --only-installed --data name > $toolsDir/tinytex-pkg-installed-$(Get-Date -Format ddMMyyHH).txt`""
```

Comment:
Is it okay to used a fixed filename? Imagine this script `choco/tools/chocolateybeforemodify.ps1` is executed at 11:59:59 on 2021/12/31, and the next script `choco/tools/chocolateyinstall.ps1` is executed at 00:00:05 on 2022/01/01, then you'll have trouble.

```suggestion
$installed_packages_command = "/C `"$toolsdir\TinyTeX\bin\win32\tlmgr.bat info --list --only-installed --data name > $toolsDir/tinytex-pkg-installed.txt`""
```

Reply:
I guess it's fine to do this since we anyway delete the file later. Also, the same needs to be changed in scoop.

Reply:
I think I did that in Scoop because sometimes the file is not deleted (maybe it depends on the order of things or because of early exit in install step) and in that case I wanted to use the right file. 

I knew about the limitation of pre Install start at 11:59:59 on 2021/12/31 and post install is at 00:00:05 2022/01/01 but I felt that this was really rare situation and that it would be ok. The situation I described before was happening to me during test. 

I can review though, but I remember to have done that on purpose in Scoop.

Reply:
@cderv The trouble will occur if the year, month, day, and/or hour become different. For example, if the pre install happens on 8:59 and post install happens on 9:00, it will fail to find the correct file. I think the chance for the hour to change is much higher than the example I gave above.

Anyway, we can revisit this issue if anyone runs into this problem you described (i.e., file not deleted).